config/initializers/constants.rb

```diff
@@ -908,7 +908,7 @@ class Constants
   IMPORT_REPOSITORY_ITEMS_LIMIT = 2000
 
   # Very basic regex to check for validity of emails
-  BASIC_EMAIL_REGEX = URI::MailTo::EMAIL_REGEXP
+  BASIC_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
```

Comment:
Not sure about this regex. Maybe we still need keep support of local domains like user@lan

Reply:
Revert